docs/content/documentation/systems/Kernels/index.md

```diff
@@ -1,7 +1,160 @@
 <!-- MOOSE Documentation Stub: Remove this when content is added. -->
 
 # Kernels System
-!parameters /Kernels
+
+A "Kernel" is a piece of physics. It can represent one or more operators or
+terms in the weak form of a partial differential equation.  With all terms on
+the left-hand-side, their sum is referred to as the "residual". The residual
+is evaluated at several integration quadrature points over the problem
+mesh/geometry. To implement your own physics in MOOSE, you create your own
+kernel by subclassing the MOOSE `Kernel` class.  In a `Kernel` subclass:
```

Comment:
`In a `Kernel` subclass:`, should be moved to the first part of the paragraph below.